src/components/Tab/tab.scss

```diff
@@ -30,7 +30,16 @@
     }
   }
 
+  .tab-title-wrapper {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+  }
```

Comment:
When it's not a side tab, the content will be centered, and items will be too close to each other. Let's add a gap between the title and the indicators for that case
![image](https://github.com/weka/weka-ui-components/assets/116369607/c8971583-eb1e-4e24-8a2a-1a01a36c94d1)
![image](https://github.com/weka/weka-ui-components/assets/116369607/1d18c5d2-171e-4d1e-a056-6132f98384ad)